Skype/AppSDK/PlatformConsiderations.md

```diff
@@ -40,7 +40,28 @@ DevicesManager audio interfaces are reduced on iOS. When headphones or a headset
 
 ### Permissions and security
```

Comment:
We discuss permissions for Android apps in the Getting Started topic. Think about moving the permissions and security section to https://github.com/OfficeDev/skype-docs/blob/master/Skype/AppSDK/GettingStarted.md